chat/base.py

```diff
@@ -317,6 +317,17 @@ def prompt_config(checkpoint_dir: Path, tokenizer: Tokenizer) -> Tuple[str, Tupl
         )
         return system_prompt, stop_tokens
 
+    if re.search(r"TinyLlama.*Chat", checkpoint_name):
+        system_prompt = (
+            "<|system|>\n"
+            "You are a friendly chatbot who always gives helpful, detailed, and polite answers.</s>\n"
```

Comment:
I also tried a longer system prompt (like we have for LLaMa 2), but it's too overwhelming for the model, and it doesn't cope with it.